packages/runner-ct/webpack.config.ts

```diff
@@ -1,101 +0,0 @@
-process.env.NO_LIVERELOAD = '1'
-
-import _ from 'lodash'
-import path from 'path'
-import type webpack from 'webpack'
-import { getCommonConfig, getCopyWebpackPlugin } from '@packages/web-config/webpack.config.base'
-import * as cyIcons from '@packages/icons'
```

Comment:
We should add this to the webpack configuration in runner/webpack. I think it actually fixes #22404

Reply:
I fixed this and learned that favicon shows when the URL port is `4455`, but it fails when `5555`.